src/main/app/claims/featureTogglesClient.ts

```diff
@@ -0,0 +1,29 @@
+import { request as requestPromiseApi, RequestPromiseAPI } from 'client/request'
+import { User } from 'idam/user'
+import * as config from 'config'
+
+export const featureTogglesApiBaseUrl: string = `${config.get<string>('feature-toggles-api.url')}`
+const featureTogglesApiUrl: string = `${featureTogglesApiBaseUrl}/api/ff4j/check/cmc_admissions`
```

Comment:
`cmc_admissions` shouldn't be part of the api url
It should be a parameter as that's the actual feature toggle